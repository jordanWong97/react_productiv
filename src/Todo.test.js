import React from "react";
import { render } from "@testing-library/react";
import Todo from "./Todo";

describe("todo component", function () {
  it("renders without crashing", function () {
    render(<Todo id={1} title="test" description="test" priority={1}/>);
  });

  it("todo div to be in the document", function () {
    const { container } =
      render(<Todo id={1} title="test" description="test" priority={1}/>);
    expect(container.querySelector(".Todo")).toBeInTheDocument();
  });

  it ("snapshot test", function () {
    const { container } =
      render(<Todo id={1} title="test" description="test" priority={1}/>);
    expect(container).toMatchSnapshot();
  });
});

