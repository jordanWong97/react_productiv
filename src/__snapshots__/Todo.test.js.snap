// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`todo component snapshot test 1`] = `
<div>
  <div
    class="Todo"
  >
    <div>
      <b>
        test
      </b>
       
      <small>
        (priority: 
        1
        )
      </small>
    </div>
    <div>
      <small>
        test
      </small>
    </div>
  </div>
</div>
`;
